blocks/woo-loyal-customer/block.php

```diff
@@ -0,0 +1,37 @@
+<?php
+// Minimum number of previous purchases to see the coupon
+$min_previous_purchases = block_value('previous-purchases');
```

Comment:
Put spaces on both sides of the opening and closing parentheses (here and elsewhere in this file)

`block_value( 'previous-purchases' )`